pkg/vex/vex.go

```diff
@@ -111,6 +114,11 @@ func New(ctx context.Context, report *types.Report, opts Options) (*Client, erro
 			} else if v == nil {
 				continue
 			}
+		case TypeSBOMReference:
+			v, err = NewSBOMReferenceSet(report)
+			if err != nil {
+				return nil, xerrors.Errorf("failed to create set of external VEX documents: %w", err)
+			}
```

Comment:
Now, we should handle nil :)

```suggestion
			v, err = NewSBOMReferenceSet(report)
			if err != nil {
				return nil, xerrors.Errorf("failed to create set of external VEX documents: %w", err)
			} else if v == nil {
				continue
			}
```

Reply:
Done

Reply:
This now raises a question for me. If this is an experimental feature and somebody explicitly scans an SBOM with `--vex sbom-ref` shouldn't trivy throw an error if it finds no valid external references in the scanned SBOM?

Similar to how it is now throwing an error if e.g. a server responds with 404.